pulsar/datadog_checks/pulsar/check.py

```diff
@@ -0,0 +1,100 @@
+            'pulsar_storage_size': 'storage.size',
+            'pulsar_storage_write_latency_count': 'storage.write_latency_count',
+            'pulsar_storage_write_latency_le_0_5': 'storage.write_latency_le_0_5',
+            'pulsar_storage_write_latency_le_1': 'storage.write_latency_le_1',
+            'pulsar_storage_write_latency_le_10': 'storage.write_latency_le_10',
+            'pulsar_storage_write_latency_le_100': 'storage.write_latency_le_100',
+            'pulsar_storage_write_latency_le_1000': 'storage.write_latency_le_1000',
+            'pulsar_storage_write_latency_le_20': 'storage.write_latency_le_20',
+            'pulsar_storage_write_latency_le_200': 'storage.write_latency_le_200',
+            'pulsar_storage_write_latency_le_5': 'storage.write_latency_le_5',
+            'pulsar_storage_write_latency_le_50': 'storage.write_latency_le_50',
+            'pulsar_storage_write_latency_overflow': 'storage.write_latency_overflow',
+            'pulsar_storage_write_latency_sum': 'storage.write_latency_sum',
+            'pulsar_storage_write_rate': 'storage.write_rate',
+            'pulsar_subscription_back_log': 'subscription.back_log',
+            'pulsar_subscription_back_log_no_delayed': 'subscription.back_log_no_delayed',
+            'pulsar_subscription_blocked_on_unacked_messages': 'subscription.blocked_on_unacked_messages',
+            'pulsar_subscription_delayed': 'subscription.delayed',
+            'pulsar_subscription_msg_rate_out': 'subscription.msg_rate_out',
+            'pulsar_subscription_msg_rate_redeliver': 'subscription.msg_rate_redeliver',
+            'pulsar_subscription_msg_throughput_out': 'subscription.msg_throughput_out',
+            'pulsar_subscription_unacked_messages': 'subscription.unacked_messages',
+            'pulsar_subscriptions_count': 'subscriptions.count',
+            'pulsar_throughput_in': 'throughput_in',
+            'pulsar_throughput_out': 'throughput_out',
+            'pulsar_topics_count': 'topics_count',
+            'scrape_duration_seconds': 'scrape_duration_seconds',
+            'scrape_samples_post_metric_relabeling': 'scrape_samples_post_metric_relabeling',
+            'scrape_samples_scraped': 'scrape_samples_scraped',
+            'topic_load_times': 'topic_load_times',
+            'topic_load_times_count': 'topic_load_times_count',
+            'topic_load_times_sum': 'topic_load_times_sum',
+            'up': 'broker.up',
+        }
+
+        instance.update(
+            {
+                'prometheus_url': url,
+                'namespace': self.NAMESPACE,
+                'metrics': [self.metrics_mapper],
+                # 'send_histograms_buckets': send_buckets,
+                # 'send_distribution_counts_as_monotonic': instance.get('send_distribution_counts_as_monotonic', True)
+                # default to True to submit _count histogram/summary as monotonic
+                # counts to Datadog
```

Comment:
```suggestion
                'send_distribution_counts_as_monotonic': instance.get('send_distribution_counts_as_monotonic', True),
                'send_distribution_sums_as_monotonic': instance.get('send_distribution_sums_as_monotonic', True),
```

Let's include and override these options to be True (which is the correct behavior) for prometheus count/sum metrics.